tests/tests_pytorch/models/test_hooks.py

```diff
@@ -762,15 +762,17 @@ def validation_step(self, x2, batch_nb2):
             self.assert_args(x2, batch_nb2)
             return super().validation_step(x2, batch_nb2)
 
-        def test_step(self, x3, batch_nb3, dl_idx3):
+        # we don't support a different name for `dataloader_idx`
```

Comment:
This is a breaking change. I don't see a simple way to support it

Reply:
I think it would go too far. I prefer to establish a clear, unambiguous contract here.